assets/scss/bootstrap-migration/custom/_tables.scss

```diff
@@ -0,0 +1,59 @@
+th {
+  &.col-sortable {
+    background-image: url(../../images/arrow-inactive-down.svg);
+    background-repeat: no-repeat;
+    background-position: calc(100% - 10px);
+  }
+
+  &.filter {
+    background-repeat: no-repeat;
+    background-position: calc(100% - 10px);
+    background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNCIgaGVpZ2h0PSIxNCIgZmlsbD0ibm9uZSIgdmlld0JveD0iMCAwIDE0IDE0Ij4KICAgIDxwYXRoIGZpbGw9IiNCQUJBQkEiIGQ9Ik0xMS41MyAySDIuNDdjLS40MTcgMC0uNjI3LjUwNS0uMzMyLjhMNS43NSA2LjQxM3Y0LjAyNGMwIC4xNTMuMDc1LjI5Ny4yLjM4NWwxLjU2MiAxLjA5M2MuMzA5LjIxNi43MzgtLjAwMy43MzgtLjM4NFY2LjQxM0wxMS44NjIgMi44Yy4yOTUtLjI5NC4wODYtLjgtLjMzMS0uOHoiLz4KPC9zdmc+Cg==');
+  }
+
+  &.desc,
+  &.asc,
+  &.filtered,
+  &.filter:hover {
+    background-repeat: no-repeat;
+    background-position: calc(100% - 10px);
+  }
+
+  &.desc {
+    background-image: url(../../images/arrow-active-down.svg);
+  }
+
+  &.asc {
+    background-image: url(../../images/arrow-active-up.svg);
+  }
+
+  &.filtered {
+    background-image: url(../../images/filter-active.svg);
+  }
+
+  &.filter:hover {
+    background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNCIgaGVpZ2h0PSIxNCIgZmlsbD0ibm9uZSIgdmlld0JveD0iMCAwIDE0IDE0Ij4KICAgIDxwYXRoIGZpbGw9IiNCQUJBQkEiIGQ9Ik0xMS41MyAySDIuNDdjLS40MTcgMC0uNjI3LjUwNS0uMzMyLjhMNS43NSA2LjQxM3Y0LjAyNGMwIC4xNTMuMDc1LjI5Ny4yLjM4NWwxLjU2MiAxLjA5M2MuMzA5LjIxNi43MzgtLjAwMy43MzgtLjM4NFY2LjQxM0wxMS44NjIgMi44Yy4yOTUtLjI5NC4wODYtLjgtLjMzMS0uOHoiLz4KPC9zdmc+Cg==');
+  }
+}
+
+[ng-transclude='toggleDropdown'] {
+  width: 100%;
+}
+
+[ng-dropdown-multiselect] > div > ul > li.divider {
+  margin: 0.563rem 0;
+  height: 1px;
+  overflow: hidden;
+  background-color: $gray-200;
+}
+
+[ng-dropdown-multiselect] > div > ul > li > a:focus {
+  background: none;
+}
```

Comment:
Don't use base selectors for specific case, incapsulate it into a page/group/component class instead

Reply:
This element is used only once